app.js

```diff
@@ -70,7 +70,7 @@ app.post('/api/v1/projects', async (req, res) => {
   for (let requiredParam of ['name']) {
     if (!newProject[requiredParam]) {
       return res.status(422).send({
-        error: `Required paramete of ${requiredParam} is missing from request.`
+        error: `Required parameter of "${requiredParam}" is missing from request.`
```

Comment:
I think we need to add logic/error handling for checking if the project name already exists.